src/academy/devdojo/maratonajava/javacore/streams/domain/LightNovel.java

```diff
@@ -5,17 +5,25 @@
 public class LightNovel {
     private String title;
     private double price;
+    private Category category;
 
     public LightNovel(String title, double price) {
         this.title = title;
         this.price = price;
     }
 
+    public LightNovel(String title, double price, Category category) {
```

Comment:
You can also use **@AllArgsConstructor** from the Lombok project.

Reply:
Yes, but in this case I need to follow the course structure.